keel-core/src/test/kotlin/com/netflix/spinnaker/keel/enforcers/EnvironmentExclusionEnforcerTest.kt

```diff
@@ -30,6 +31,7 @@ internal class EnvironmentExclusionEnforcerTest {
     }
   }
 
+  @kotlinx.coroutines.ExperimentalCoroutinesApi
```

Comment:
This annotation was added to suppress a warning because I'm using the `runBlockingTest` construct in this test.